client/components/segmented-control/item.jsx

```diff
@@ -29,11 +29,16 @@ var SegmentedControlItem = React.createClass( {
 			'is-selected': this.props.selected
 		} );
 
+		var linkClassName = classNames( {
+			'segmented-control__link': true,
```

Comment:
Since this is always true, you could shorten this to:

```js
const linkClassName = classNames( 'segmented-control__link', {
	[ `item-index-${ this.props.index }` ]: this.props.index != null,
} );
```

Also, no `var`s please :-)